src/main/java/org/jabref/gui/JabRefDialogService.java

```diff
@@ -424,25 +424,23 @@ public void notify(String message) {
         //       The event log is not that user friendly (different purpose).
         LOGGER.info(message);
 
-        UiTaskExecutor.runInJavaFXThread(() -> {
-            Notifications.create()
-                         .text(message)
-                         .position(Pos.BOTTOM_CENTER)
-                         .hideAfter(TOAST_MESSAGE_DISPLAY_TIME)
-                         .owner(mainWindow)
-                         .threshold(5,
-                                 Notifications.create()
-                                              .title(Localization.lang("Last notification"))
-                                              .text(
-                                                    "(" + Localization.lang("Check the event log to see all notifications") + ")"
-                                                     + "\n\n" + message)
-                                              .onAction(e -> {
-                                                     ErrorConsoleAction ec = new ErrorConsoleAction();
-                                                     ec.execute();
-                                                 }))
-                         .hideCloseButton()
-                         .show();
-        });
+        UiTaskExecutor.runInJavaFXThread(() -> Notifications.create()
+                                                        .text(message)
+                                                        .position(Pos.BOTTOM_CENTER)
+                                                        .hideAfter(TOAST_MESSAGE_DISPLAY_TIME)
+                                                        .owner(mainWindow)
+                                                        .threshold(5,
+                             Notifications.create()
```

Comment:
This new indent is a bit strange - maybe keep the old indent - on line 428 it also "felt" better, because it is indented 4 spaces, not 30 or so ^^